source/use-cases/sensitive-data-encryption.txt

```diff
@@ -57,12 +57,127 @@ cubilia Curae; Nunc non interdum purus, ultricies laoreet tortor.
 B. Define a JSON Schema
 ~~~~~~~~~~~~~~~~~~~~~~~
 
-Proin non mi a felis luctus vulputate nec pharetra odio. Ut pretium
-scelerisque nulla, sed euismod nisi pellentesque sit amet. Proin mollis
-mauris eu libero ullamcorper rutrum. Pellentesque vitae enim sed magna
-accumsan maximus et quis mi. Ut vel laoreet turpis. Vivamus vitae odio
-sed arcu finibus pretium vel maximus nisi. Nullam gravida consequat
-porta.
+An important concept in database interaction is `data validation 
+<https://docs.mongodb.com/manual/core/schema-validation/>`_. MongoDB 
+works to make this as simple as possible, using 
+`JSON Schema <https://docs.mongodb.com/manual/reference/operator/query/jsonSchema/#json-schema>`_ 
+for validation during insert and update operations. Within the context of Field Level Encryption,
+JSON Schema is used to specify several configuration details, allowing the drivers
+to automatically encrypt and decrypt fields. 
+
+* To do so, you need to specify two things in JSON Schema for each encrypted field:
+    a) The encryption type (deterministic or random)
+    b) The data key
+    c) The BSON Type
+* You can use a single data key for all encrypted fields or you can use individual data keys that are specific to each field.
```

Comment:
I was asking whether line 72 was meant to be part of the bulleted list (ll. 69-71) because the outer list (ll. 68, 72) does not have a unifying theme. My suggestion was to provide a unifying theme "there are several decisions...". This could be accompanied by a description of the table and what the JSON Schema column names are before line 79.

Reply:
I've switched the second bullet to a note and clarified the first bullet (nested list). I've updated the staging page. Let me know what you think @ccho-mongodb

Reply:
Since this is a single bullet, could it just be a top-level item? Also, I'm not sure if you intended to do this, but the formatting came out in bold from what I see on staging.
![Screen Shot 2019-09-03 at 4 29 51 PM](https://user-images.githubusercontent.com/52428683/64206312-13770a80-ce68-11e9-93dc-f524f767b340.png)

Reply:
Fixed, I removed the bold first bullet and made the rest of the bullets top-level; it now reads:

While designing your database, you may want to enforce particular datatypes in specific fields. MongoDB uses JSON Schema to reduce the complexity of data validation during insert and update operations. Client-Side Field Level Encryption with MongoDB requires the use of JSON Schema for automatic encryption. Encrypted fields require specifying the following configuration options:

The encryption algorithm (deterministic or random)
The data key
The BSON Type

Is this okay? @ccho-mongodb

Reply:
Are you okay w/ resolving this discussion if you feel your concerns are addressed? I ask because it's easier for me to see what's left to fix in a PR when the only remaining unresolved comments are stuff I have yet to fix 

@ccho-mongodb